Deploy.md

```diff
@@ -30,5 +30,5 @@ npx hardhat verify --contract contracts/token/RarePizzasBox.sol:RarePizzasBox --
 
 ## Transfer ownable
 
- npx hardhat run --network rinkeby scripts/box.transfer.ownable.ts
+ npx hardhat run --network mainnet scripts/box.transfer.ownable.ts
```

Comment:
perhaps break into 2 entries? 1 for testnet, 1 for mainnet

Reply:
the process im following is to list them as rinkeby until the deployment happens and then change them to mainnet.  this wont be necessary when we have ci